Controller/DashboardAdminController.php

```diff
@@ -38,6 +38,8 @@ public function composeAction(Request $request = null)
             throw new AccessDeniedException();
         }
 
+        $template = $this->admin->getTemplate('compose');
```

Comment:
Don't use One-Time variables and put them faaaar from where they are used unless you have a good reason.